src/olympia/reviews/helpers.py

```diff
@@ -91,6 +93,15 @@ def user_can_delete_review(request, review):
             acl.action_allowed(request, 'Addons', 'Edit')))
 
 
+class CanDeleteReviewPermission(BasePermission):
+    """A DRF permission class wrapping user_can_delete_review()."""
+    def has_permission(self, request, view):
+        return self.request.user.is_authenticated()
+
+    def has_object_permission(self, request, view, obj):
+        return user_can_delete_review(request, obj)
+
```

Comment:
Yeah, I was lazy but it makes more sense in `reviews/permissions.py`. I'll also add unit tests for it.